app/services/people/nomis_profile_validator.rb

```diff
@@ -6,8 +6,8 @@ class NomisProfileValidator
 
     validates :latest_nomis_booking_id, presence: true
 
-    def initialize(profile)
-      @latest_nomis_booking_id = profile.latest_nomis_booking_id
+    def initialize(person)
```

Comment:
Maybe this class should be renamed to `NomisPersonValidator` as it is no longer validating a profile

Reply:
I do agree. Alessandro mentioned this during pair review. I'll rename it and rerun the tests.